mmv1/products/databasemigrationservice/MigrationJob.yaml

```diff
@@ -157,6 +157,7 @@ properties:
         output: true
         item_type:
           type: KeyValuePairs
+          custom_flatten: 'templates/terraform/custom_flatten/dms_migration_job_error_details.tmpl'
```

Comment:
it looks like a custom_flatten at this level isn't honored - there's no diff in the providers - so you'll need to move it up into the main `details` field & modify the code to handle the fact that it's a list of maps (instead of an individual map)